locmap/docs/admin.md

```diff
@@ -0,0 +1,60 @@
+#Admin Resources
+
+##Description
+Administrative API calls
+
+***
+##Requires Admin Authentication
```

Comment:
Any particular reason why this (and the equivalent in user.md) is a header? It looks a bit odd to me.